proc_windows.go

```diff
@@ -20,7 +21,7 @@ func spawnProc(proc string, errCh chan<- error) {
 	cmd.Stdout = logger
 	cmd.Stderr = logger
 	cmd.SysProcAttr = &windows.SysProcAttr{
-		CreationFlags: windows.CREATE_UNICODE_ENVIRONMENT | 0x00000200,
+		CreationFlags: windows.CREATE_UNICODE_ENVIRONMENT | syscall.CREATE_NEW_PROCESS_GROUP,
```

Comment:
Maybe `windows.CREATE_NEW_PROCESS_GROUP` instead? The latter will be guarded with build tags.